budgetportal/tests/test_prov_infra_projects.py

```diff
@@ -31,12 +39,16 @@ def setUp(self):
         EmailAddress.objects.create(user=user, email=EMAIL, verified=True)
         self.path = os.path.dirname(__file__)
         self.financial_year = FinancialYear.objects.create(slug="2019-20")
-        self.timeout = 10
-
-        super(ProvInfraProjectSeleniumTestCase, self).setUp()
-
-    def test_upload_xlsx_for_prov_infra_projects(self):
-        filename = "budgetportal/tests/test_data/test_import_prov_infra_projects.xlsx"
+        self.quarter = Quarter.objects.create(number=1)
+        self.timeout = 60
+        self.search_url = "/infrastructure-projects/provincial/"
+        super(ProvInfraProjectSeleniumIRMSnapshotTestCase, self).setUp()
+
+    def test_import_irm_snapshot(self):
```

Comment:
Tis doesn't need to be a selenium test - it's fine for it just to do IRMSnapshot.create(...) and long as it ends up querying the API

Reply:
But it's also fine as a selenium test

Reply:
I think I'll change it to a normal test because Selenium fails even though everything seems correct.

Reply:
Test changed.

https://github.com/vulekamali/datamanager/pull/213/commits/1d97757deae6ca317321058181f49a504cb5615c